### PR TITLE
dont write a certificate bundle if the shipped ca bundle is empty

### PR DESCRIPTION
**Pull request by icewind1991**
For #2910

Instead of breaking, we log

### CODE REVIEW

File: lib/private/Security/CertificateManager.php [hunk 1]
```diff
@@ -104,6 +112,13 @@ public function createCertificateBundle() {
 			$this->view->mkdir($path);
 		}
 
+		$defaultCertificates = file_get_contents(\OC::$SERVERROOT . '/resources/config/ca-bundle.crt');
+		if (strlen($defaultCertificates) < 1024) { // sanity check to verify that we have some content for our bundle
+			// log as expected so we have a stacktrace
```
Comment (MorrisJobke) [inline lib/private/Security/CertificateManager.php]: `s/expected/exception/`?

**Review by MorrisJobke — CHANGES_REQUESTED** [review]

File: lib/private/Security/CertificateManager.php (hunk 1 above)
Comment (icewind1991) [inline lib/private/Security/CertificateManager.php]: fixed

**Comment by LukasReschke** [comment]
Tests do fail.

**Comment by icewind1991** [comment]
@LukasReschke fixed

**Review by rullzer — APPROVED** [review]

**Pull request merged**